cmake/BLTPrivateMacros.cmake

```diff
@@ -589,6 +592,8 @@ macro(blt_split_source_list_by_language)
             message(FATAL_ERROR "blt_split_source_list_by_language given source file with no extension: ${_file}")
         endif()
 
+        get_filename_component(_name ${_file} NAME)  
```

Comment:
Not sure if this syntax is worth flagging, but I noticed that `${_file}` isn't quoted in parts of `blt_split_source_by_language`, which gives me the impression this macro may not yield the desired results on paths that include spaces.

Reply:
#422 added.  Will check later, thanks for the catch.